### ISSUE TITLE
Strange work of the standard OCR tool

### ISSUE THREAD

**Issue by TaunT** [ogkalu2/comic-translate#153]
This is probably a life hack =)

I usually translate comics from English, and there were always a lot of mistakes when digitizing. Yesterday I translated a page from French and forgot to switch back, today I started digitizing from English and everything went perfectly, not a single mistake.

I checked - the result is constant. If you select French when digitizing English text, the result is much better!)

**Comment by ogkalu2** [comment]
No what I mean is, what OCR option do you have in settings when this happens? Default, Microsoft or Google ?

**Comment by TaunT** [comment]
Ah, it all turned out to be very simple) For English, the simplest model is used, and for French, GPT-4o

### OCR
By Default:
* [EasyOCR](https://github.com/JaidedAI/EasyOCR) for English
* [GPT-4o](https://platform.openai.com/docs/guides/vision) for French, Russian, German, Dutch, Spanish and Italian. Paid, Requires an API Key.

**Comment by TaunT** [comment]
@ogkalu2 
> No what I mean is, what OCR option do you have in settings when this happens? Default, Microsoft or Google ?
I didn't understand at first) Yes, it is Default in the settings

**Comment by ogkalu2** [comment]
Yes that's it. Pororo also supports English. I think it's better also. Maybe I should switch to that for Default ?

Can you set your source language to Korean and see if the English results are better than EasyOCR ?

**Comment by TaunT** [comment]
in "Korean" mode some words were not recognized and replaced with hieroglyphs. 
![image](https://github.com/user-attachments/assets/3f73acdf-dd56-4355-9bab-7446f6d22a1c)

It is necessary to try to change the default model in the code.

 By the way, with GPT there is also a problem of intersecting blocks

**Comment by TaunT** [comment]
@ogkalu2  Would it be nice to add the choice of GPT to the OCR settings?

**Comment by ogkalu2** [comment]
> in "Korean" mode some words were not recognized and replaced with hieroglyphs. ![image](https://private-user-images.githubusercontent.com/1741557/375477995-3f73acdf-dd56-4355-9bab-7446f6d22a1c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzIwNzE3MzYsIm5iZiI6MTczMjA3MTQzNiwicGF0aCI6Ii8xNzQxNTU3LzM3NTQ3Nzk5NS0zZjczYWNkZi1kZDU2LTQzNTUtOWJhYi03NDQ2ZjZkMjJhMWMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEyMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMjBUMDI1NzE2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZDAyODQxNzdkYzA5MTU4YzQzNzhkYmYzMzkyOWEwMGFmMWMzY2E4MDllYjZlN2M5YTBjOGFjNmU0NGIwNDA2NyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.vw8-IaJkkSc1iBiul8BRmLo__eLjnR2sNCwwyiDRovU)
> 
> It is necessary to try to change the default model in the code.
> 
> By the way, with GPT there is also fix a problem of intersecting blocks

In modules/ocr, line 203, change 
self.pororo_cache = PororoOcr()
to
self.pororo_cache = PororoOcr(lang='en') and see if it's better

What do you mean by the problem of intersecting blocks ?

**Comment by ogkalu2** [comment]
@TaunT 
I've added GPT to the list of OCR models but i've also changed the English default to doctr. It's not as good as GPT but it's much better than EasyOCR
